test/gtest/uct/uct_test.h

```diff
@@ -153,6 +153,7 @@ std::ostream& operator<<(std::ostream& os, const resource* resource);
     UCT_TEST_IB_TLS, \
     ugni, \
     mm, \
```

Comment:
could you please update "transport" batch file of ucx_perftest w/ "cma" and probably "mm"?
it will be part of jenkins routing than.